Autoload/Network.gd

```diff
@@ -0,0 +1,50 @@
+extends Node
+
+var port = 1024
+var max_clients = 3
+
+var server = null
+var client = null
+
+var ip_address = ""
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	for ip in IP.get_local_addresses():
+		if ip.begins_with("192.168.") or ip.begins_with("10.") or ip.begins_with("172.16") and not ip.ends_with(".1"):
```

Comment:
Thoses are pretty specific ip subnets... probably want to move them to a config.
Also, don't you want to exit the loop after finding an acceptable ip?

Reply:
Good point, I'll add a break in the if-statement